host/containerinit/init.go

```diff
@@ -535,9 +619,6 @@ func Main() {
 		log.Fatalf("Unable to unmarshal config: %v", err)
 	}
 
-	// Propagate the plugin-specific container env variable
-	config.Env["container"] = os.Getenv("container")
-
```

Comment:
Yes, I figured that `os.Getenv("container") == ""`, since I couldn't find anything setting `container`